stdlib/public/SwiftShims/RefCount.h

```diff
@@ -92,24 +100,60 @@ class StrongRefCount {
 
   // Increment the reference count.
   void increment() {
+#if defined(COMPILER_HAS_ATOMIC_INTRINSICS)
     __atomic_fetch_add(&refCount, RC_ONE, __ATOMIC_RELAXED);
+#else
+    std::atomic<uint32_t> stdRefCount = ATOMIC_VAR_INIT(refCount);
```

Comment:
I'm fairly sure there might be some bugs in this, but I wanted to get the thing building. I think the issue you mentioned is correct - perhaps we then need to set `refCount` to the value of `stdRefCount` here.
Either way, take a look at some of the TODOs I've sprinkled in the file. I'm not sure what the fix is for the issue that there is no such `std::atomic_sub_fetch_explicit` even though there is a `__atomic_sub_fetch` intrinsic with Clang/GCC